sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -80,3 +80,20 @@ Our cloud disk backend is based on WebSocket and distributed across a global CDN
 While v86 might use an open proxy to the internet, this approach is not feasible for CheerpX due to its scale of use. An open proxy can pose significant security and abuse risks when used at scale.
 
 Instead, CheerpX supports Tailscale-based networking, which can accommodate many use cases. It's important to note that developers using CheerpX are responsible for implementing appropriate security measures and preventing potential abuse.
+
+## Missing cross-origin isolation
+
+If you encounter the following error message:
+
+`Uncaught CheerpX initialization failed: DataCloneError: DedicatedWorkerGlobalScope.postMessage: The SharedArrayBuffer object cannot be serialized. The Cross-Origin-Opener-Policy and Cross-Origin-Embedder-Policy HTTP headers can be used to enable this.`
+
+This error occurs because CheerpX relies on the [SharedArrayBuffer], which necessitates the site to be cross-origin isolated. To activate cross-origin isolation, ensure your site is served over HTTPS and include the following headers in your responses:
```

Comment:
```suggestion
This error occurs because CheerpX relies on [SharedArrayBuffer], which require the site to be cross-origin isolated. To activate cross-origin isolation, ensure your site is served over HTTPS and include the following headers in your responses:
```